conjur/cli.py

```diff
@@ -114,10 +153,16 @@ def run(self, *args):
         if args.verbose is True:
             args.debug = True
 
-        Cli.run_client_action(resource, args)
-
-        # Explicit exit (required for tests)
-        sys.exit(0)
+        try:
+            Cli.run_client_action(resource, args)
+        except FileNotFoundError as not_found_error:
+            print("Error: No such file or directory: '%s'" % not_found_error.filename)
+            sys.exit(1)
+        except requests.exceptions.HTTPError as http_error:
+            print(str(http_error))
```

Comment:
should the logger be a singleton used across all classes or it doesn't matter?

Reply:
Capturing in a logger would make the output look like:
`2020-11-29 10:54:19,112 INFO: 404 Client Error: Not Found for url: https://localhost/secrets/myorg/variable/df`

but I want to print to stdout to avoid the `2020-11-29 10:54:19,112 INFO:` messiness.  An idea though would be add a logger to also capture this information so good feedback.

Instead of `print` would  `sys.stderr.write` be better?

Reply:
https://docs.python.org/3/howto/logging-cookbook.html
if you do 
`formatter = logging.Formatter('%(message)s')` 
i think it wll print it.

Anyway if you do sys.stderr.write its good enough too. print is bad habit in Production code ( i don't ramembar why)

Reply:
I don't think its matter if singelton or not but give same logger name in all code files so it will use the same configuration

Reply:
So you can have the decision if you use logging or not. If it is just for printing errors to stderr so `sys.stderr.write` is good enough i think